src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,120 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+
+const Notification = ({ toast, message, body, status, onClose }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  const {
+    gap: containerGap,
+    elevation: containerElevation,
+    round: containerRound,
+  } = theme.notification.container;
+
+  const { pad: iconContainerPad, round: iconContainerRound } =
+    theme.notification.iconContainer;
+
+  const { pad: textContainerPad } = theme.notification.textContainer;
+
+  const { size: messageTextSize, weight: messageTextWeight } =
+    theme.notification.messageText;
+
+  const { size: bodyTextSize } = theme.notification.bodyText;
+
+  const { margin: buttonMargin } = theme.notification.button;
+
+  const Icons = {
+    StatusGood: theme.notification.icons.statusGood,
+    StatusWarning: theme.notification.icons.statusWarning,
+    StatusUnknown: theme.notification.icons.statusUnknown,
+    FormClose: theme.notification.icons.closeButton,
+  };
+
+  const iconColor = theme.notification.icons.color;
+
+  let icon;
+  let color;
+  switch (status) {
+    case 'warning':
+      color = theme.notification.status.warning.color;
+      icon = <Icons.StatusWarning color={iconColor} />;
+      break;
+    case 'good':
+      color = theme.notification.status.good.color;
+      icon = <Icons.StatusGood color={iconColor} />;
+      break;
+    case 'unknown':
+      color = theme.notification.status.unknown.color;
+      icon = <Icons.StatusUnknown color={iconColor} />;
+      break;
+    default:
+      color = theme.notification.status.unknown.color;
+      icon = <Icons.StatusUnknown color={iconColor} />;
+      break;
+  }
+
+  let content = (
+    <>
+      <Box
+        round={iconContainerRound}
+        background={color}
+        pad={iconContainerPad}
+        justify="center"
```

Comment:
I don't think we need `justify` here.

Reply:
without `justify="center"`, the icon is not centered to the content. see attached screen shots
<img width="345" alt="Screen Shot 2021-07-28 at 4 01 07 PM" src="https://user-images.githubusercontent.com/38971752/127388021-8e5c4494-4870-4f64-8ddc-58910da60262.png">
<img width="510" alt="Screen Shot 2021-07-28 at 4 00 59 PM" src="https://user-images.githubusercontent.com/38971752/127388023-fa3f4e88-0491-4690-8ed2-d3db01b78e83.png">